sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/baseSender.ts

```diff
@@ -199,6 +199,9 @@ export abstract class BaseSender {
   private async persist(envelopes: unknown[]): Promise<ExportResult> {
     try {
       const success = await this.persister.push(envelopes);
+      if (!success) {
```

Comment:
You will tons of false positives for disabled persister when disableOfflineStorage is set, is not actually failing to persist as it was configured off explicitly

Reply:
I believe the correct place to count the write failure is in the catch below

Reply:
Thanks for the catch. Yeah, since we return a promise from the `push` method in the event that the storage setting is turned off, we should only need to collect true failures from the catch block. Updated.